MentorBook.Data/Repositories/Interfaces/IFriendshipRepsoitory.cs

```diff
@@ -0,0 +1,11 @@
+using MentorBook.Data.Models;
+using MentorBook.Data.Repositories;
+using System.Collections.Generic;
+
+namespace MentorBook.Data.Repositories
+{
+    public interface IFriendshipRepository
```

Comment:
Same like the service, it's better to have the interface with identical name like the implementation